src/main/resources/application.yml

```diff
@@ -20,3 +20,13 @@ spring:
     properties:
       mail.smtp.auth: true
       mail.smtp.starttls.enable: true
+
+  url: "https://countries-cities.p.rapidapi.com/location/country/RS/city/list?page=1&per_page=100&population=1501"
```

Comment:
If this is the value of the URL, are you sure that `url.replace("{1}", ii) `will return expected value?

Reply:
Checked what was the problem, now its done